op-batcher/metrics/metrics.go

```diff
@@ -278,14 +293,14 @@ func (m *Metrics) RecordChannelClosed(id derive.ChannelID, numPendingBlocks int,
 }
 
 func (m *Metrics) RecordL2BlockInPendingQueue(block *types.Block) {
-	size := float64(estimateBatchSize(block))
-	m.pendingBlocksBytesTotal.Add(size)
-	m.pendingBlocksBytesCurrent.Add(size)
+	size := estimateBatchSize(block)
+	m.pendingBlocksBytesTotal.Add(float64(size))
+	atomic.AddInt64(&m.pendingBytes, int64(size))
 }
 
 func (m *Metrics) RecordL2BlockInChannel(block *types.Block) {
-	size := float64(estimateBatchSize(block))
-	m.pendingBlocksBytesCurrent.Add(-1 * size)
```

Comment:
this metric is no longer incremented... any reason this was removed?

Reply:
I converted that metric to type GaugeFunc backed by m.pendingBytes, so you can't increment the metric directly, but it's indirectly updated by the atomic update of m.pendingBytes.

Reply:
thanks, TIL